apps/modernization-ui/src/apps/page-builder/page/management/header/PageHeader.tsx

```diff
@@ -21,7 +23,12 @@ const PageHeader = ({ page, tabs, onAddTabSuccess, children }: PageHeaderProps)
                 </div>
                 {children}
             </div>
-            <PageTabs pageId={page.id} tabs={tabs} onAddSuccess={onAddTabSuccess} />
+            <DragDropProvider
+                pageData={page}
+                currentTab={page.tabs?.findIndex((x: PagesTab) => x.name === selected?.name) ?? 0}
+                successCallBack={onAddTabSuccess}>
+                <PageTabs pageId={page.id} tabs={tabs} onAddSuccess={onAddTabSuccess} />
+            </DragDropProvider>
```

Comment:
Question: It looks like the `PageTabs` component doesn't do anything with the `DragDropProvider`, but the contained `ManageTabs` component does. Can we move this provider to inside the `PageTabs` and just wrap the component that utilizes it? 

I might be overlooking something simple

Reply:
The problem is I need the tab and page info that's defined in PageHeader--I thought it would be cleaner to put it here than to prop drill this down another level.

Reply:
I see, one option would be to create a wrapping component around `DragDropProvider`, such as `PageDragDropProvider` (or some better name).

This wrapper could use the `usePageManagement` hook to grab the current `page` and `selected` values.

This is optional and up to you.